uint8  drive_motor_id
int16  drive_motor_speed
uint32 encoding_count